src/components/common/UIComponents.jsx

```
@@ -13,34 +14,42 @@ import PackageOpen from 'lucide-react/dist/esm/icons/package-open';
+    cyan:     'oklch(68% 0.12 200)',
+    early:    'oklch(64% 0.20 142)',
+    mid:      'oklch(76% 0.16 68)',
+    late:     'oklch(62% 0.22 25)',
+    nether:   'oklch(60% 0.20 15)',
+    end:      'oklch(65% 0.15 290)',
+    extreme:  'oklch(66% 0.20 45)',
+    desc:     'oklch(68% 0.12 200)',
+};
+
+// Compatibility alias — consumed by components not yet redesigned
 export const COLORS = {
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)